labcodes/labstorybook
# DSYS-179: Fix prop types

**Link to task**
https://labcodes.atlassian.net/browse/DSYS-178

**Description**
This PR replaces the deafult props of all props of type _function_. Instead of `undefined` they are now set to `() => {}`. This change led to a required small refactoring in `AbstractTextInput` regarding it's subcomponent `TrailingIcon`. Snapshots are slightly updated to reflect changes on these components.  

**How to test**
With the story book running, go to the `Text Input` section. In the playground, _disable_ the input then click on the trailing icon of `PasswordInput`. Check if nothing happens.
